src/saga/adaptors/torque/torquejob.py

```diff
@@ -867,17 +821,15 @@ def _job_get_info(self, job_id, reconnect):
                     val = val.strip()
 
                     # The ubiquitous job state
-                    if key in ['job_state']: # PBS Pro and TORQUE
-                        job_info['state'] = _torque_to_saga_jobstate(val)
+                    if key in ['job_state']:
+                        job_state = val
 
```

Comment:
This diff looks wrong, at least a little.  For one, that means that the fixed `_torque_to_saga_jobstate` it not used at all, and also, the line 873 then assigns the native state to the SAGA state.  Should the mapping move to there?